src/test/java/com/epam/tc/nitcenkov/hw3/ex1/IndexPageTest.java

```diff
@@ -0,0 +1,70 @@
+package com.epam.tc.nitcenkov.hw3.ex1;
+
+import com.epam.tc.nitcenkov.hw3.BaseTest;
+import com.epam.tc.nitcenkov.hw3.pages.IndexPage;
+import com.epam.tc.nitcenkov.hw3.pages.components.HeaderComponent;
+import com.epam.tc.nitcenkov.hw3.pages.components.LeftSectionComponent;
+import com.epam.tc.nitcenkov.hw3.pages.components.LoginComponent;
+import com.epam.tc.nitcenkov.hw3.util.Util;
+import java.util.Arrays;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.annotations.Test;
+
+public class IndexPageTest extends BaseTest {
+
+    SoftAssertions softAssertions = new SoftAssertions();
+
+    @Test
+    public void firstExerciseTest() {
+
+        //Assert Browser title
+        softAssertions.assertThat(webDriver.getTitle()).isEqualTo("Home Page");
+
+        //Perform login
+        HeaderComponent headerComponent = PageFactory.initElements(webDriver, HeaderComponent.class);
+        headerComponent.clickOnUserIcon();
+        LoginComponent loginComponent = PageFactory.initElements(webDriver, LoginComponent.class);
+        loginComponent.performLogin(Util.LOGIN, Util.PASSWORD);
```

Comment:
Please pass these values (as well as expected values) as test-method arguments, use data provider to achieve it